src/probabilities_estimators/permutation_ordinal/spatial_permutation.jl

```diff
@@ -10,23 +10,50 @@ The input data `x` are high-dimensional arrays, for example 2D arrays [^Ribeiro2
 `x` is given because we need to know its size for optimization and bound checking.
 
 A _stencil_ defines what local area around each pixel to
-consider, and compute the ordinal pattern within the stencil. Stencils are given as
-vectors of `CartesianIndex` which encode the _offsets_ of the pixes to include in the
-stencil, with respect to the current pixel. For example
-
-```julia
-data = [rand(50, 50) for _ in 1:50]
-x = data[1] # first "time slice" of a spatial system evolution
-stencil = CartesianIndex.([(0,1), (1,1), (1,0)])
-est = SpatialSymbolicPermutation(stencil, x)
-```
-
-Here the stencil creates a 2x2 square extending to the bottom and right of the pixel
-(directions here correspond to the way Julia prints matrices by default).
-Notice that no offset (meaning the pixel itself) is always included automatically.
-The length of the stencil decides the order of the permutation entropy, and the ordering
-within the stencil dictates the order that pixels are compared with.
-The pixel without any offset is always first in the order.
+consider, and compute the ordinal pattern within the stencil.
+Stencils are passed in one of the following three ways:
+
+1. as vectors of `CartesianIndex` which encode the pixels to include in the
+    stencil, with respect to the current pixel, or integer arrays of the same dimensionality
+    as the data. For example
+
+    ```julia
+    data = [rand(50, 50) for _ in 1:50]
+    x = data[1] # first "time slice" of a spatial system evolution
+    stencil = CartesianIndex.([(0,0), (0,1), (1,1), (1,0)])
+    est = SpatialSymbolicPermutation(stencil, x)
+    ```
+    Don't forget to include 0 offset if you want to include the point itself, 
+    which is almost always the case.
+    Here the stencil creates a 2x2 square extending to the bottom and right of the pixel
+    (directions here correspond to the way Julia prints matrices by default).
+    The length of the stencil decides the order of the permutation entropy, and the ordering
+    within the stencil dictates the order that pixels are compared with.
+    The pixel without any offset is always first in the order.
+
+2. as a `D`-dimensional array (where `D` matches the dimensionality of the input data)
+    containing `0`s and `1`s, where every pixel with a one is included.
+    To generate the same estimator as in 1., for example
+
+    ```julia
+    data = [rand(50, 50) for _ in 1:50]
+    x = data[1] # first "time slice" of a spatial system evolution
+    stencil = [1 1; 1 1]
+    est = SpatialSymbolicPermutation(stencil, x)
+    ```
+
+3. as a `Tuple` containing two `Tuple`s of length `D` for `D`-dimensional data.
```

Comment:
A bit more precise would be to say: *"As a `Tuple` containing two `Tuple`s, **both of length `D`**, for `D`-dimensional data"*